python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md

```diff
@@ -105,12 +107,15 @@ python llama.py
 :: to run Meta-Llama-3-8B-Instruct (LNL driver version: 32.0.101.2715)
 python llama.py --repo-id-or-model-path meta-llama/Meta-Llama-3-8B-Instruct
 
-:: to run Qwen2-1.5B-Instruct LNL driver version: 32.0.101.2715)
+:: to run Qwen2-1.5B-Instruct (LNL driver version: 32.0.101.2715)
```

Comment:
As `qwen2.py` & `qwen2.5.py` are exactly the same. We may merge them into on python scripts, for example named as `qwen2_or_2_5.py`